### PR TITLE
Rename package

### PR DESCRIPTION
**Pull request by nop33**
Related to #70

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "alephium-js",
-  "version": "1.1.2",
+  "name": "@alephium/sdk",
+  "version": "0.0.1",
```
Comment (nop33) [inline package.json:3]: Do we all agree with this version? Keep in mind that I added a message in the README to explain that breaking changes will be introduced in minor versions in contrast to the traditional semver approach (see README diff).

/cc @lf94 @mvaivre @polarker

Comment (GestioOp) [reply package.json:3]: The traceability will not be good, but I don't think it's a big deal right now (we've broken many things so far). Better start with a clean plate. So LGTM.

Comment (mvaivre) [reply package.json:3]: The traceability will not be good, but I don't think it's a big deal right now (we've broken many things so far). Better start with a clean plate. So LGTM.

File: README.md [hunk 2]
```diff
@@ -73,11 +84,12 @@ or, to watch for changes:
...
-[npm-link]: https://www.npmjs.org/package/alephium-js
+[test-badge]: https://github.com/alephium/js-sdk/actions/workflows/test.yml/badge.svg
+[test-link]: https://github.com/alephium/js-sdk/actions/workflows/test.yml
+[coverage-badge]: https://codecov.io/gh/alephium/js-sdk/branch/master/graph/badge.svg
+[coverage-link]: https://codecov.io/gh/alephium/js-sdk
+[npm-badge]: https://img.shields.io/npm/v/@alephium/sdk.svg
+[npm-link]: https://www.npmjs.org/package/@alephium/sdk
 [prettier-badge]: https://img.shields.io/badge/code_style-prettier-ff69b4.svg
 [prettier-link]: https://github.com/prettier/prettier
+[release-notes]: https://github.com/alephium/alephium-js/releases
```
Comment (mvaivre) [inline README.md:95]: Wrong link

Comment (nop33) [reply README.md]: fixed

**Review by mvaivre — APPROVED** [review]

**Comment by polarker** [comment]
I will be assisting you guys to setup the organization account. I have upgraded the personal account `alephium` to orgnazation. I will let you guys to figure out the rest, as I am not familiar with npm organization.

**Pull request merged**